README.md

```diff
@@ -1,5 +1,6 @@
 # llcpython
 # committing to the main branch as a test
+#this is a comment
```

Comment:
this needs to be updated to be capititalized

Reply:
you're right, i will do that

Reply:
just made the change @cstolitzka